packages/design-system-tailwind-preset/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.4.0]
+
+### Added
+
+- Added "accent" colors for light and dark themes ([#534](https://github.com/MetaMask/metamask-design-system/pull/534))
+
```

Comment:
Adding accent colors as Tailwind class names is added functionality, which qualifies as a minor bump. Since this package is in beta, even if it were a breaking change, we would still only bump the minor version.